### ISSUE TITLE
source-map-generator needs to be bundled

### ISSUE THREAD

**Issue by hildjj** [peggyjs/peggy#223]
1. Create a new directory.
2. `npm init`
3. `npm i github:peggyjs/peggy#1.3`
4. `npx peggy -h`

```
Error: Cannot find module 'source-map-generator'
Require stack:
- <dir>/node_modules/peggy/lib/compiler/passes/generate-js.js
- <dir>/node_modules/peggy/lib/compiler/index.js
- <dir>/node_modules/peggy/lib/peg.js
- <dir>/node_modules/peggy/bin/peggy.js
```

I think it's already bundled into `browser/peggy.min.js`.  It's not enough to use rollup to get it into the binary, since it will also be needed by the API.

**Comment by hildjj** [comment]
This will be the first full runtime dependency, so I want to make sure:
a) Nobody minds that
b) We make sure there are not entries in the docs that say "no runtime dependencies" still
c) We wouldn't rather bundle the dependency, like we do for `commander` - I don't think it would be as clean as the previous bundling without having the published library be all one .js file.
d) We think about unbundling commander and making it a runtime dependency.

**Comment by physikerwelt** [comment]
With runtime dependencies, do you mean runtime dependencies in the browser or a nodeJS dependency?
For my (test)case, it was sufficient to add the package as a dev dependency
https://gerrit.wikimedia.org/r/c/mediawiki/services/texvcjs/+/793836/2/package.json#38
and from looking at the diff of the generated source, it does not look as if there were new dependencies added.

**Comment by hildjj** [comment]
I think that for the common browser case, we will continue to bundle dependencies with rollup.  You'll still be able to do a single-line include and get things working.  For the Node case, we will unbundle `commander` from the CLI, and add it and source-map-generator as runtime dependencies.  For people that want to do their own bundling, as long as their tool does tree-shaking (to eliminate the unneeded `commander` dependency), they should get what they want from the node version.

We could also mark `commander` as optional, but I think node doesn't install optional dependencies by default anymore, so I think more people will be confused by that than leaving it as a full runtime dependency.

Does that make sense?

**Comment by hildjj** [comment]
Fixed in v2.0.0
